Tasks/Common/webdeployment-common-v2/packageUtility.ts

```diff
@@ -29,12 +29,36 @@ export class Package {
     constructor(packagePath: string) {
         this._path = PackageUtility.getPackagePath(packagePath);
         this._isMSBuildPackage = undefined;
+        let workingDirectory = tl.getVariable("system.defaultworkingdirectory");
+        try {
+            if (workingDirectory && this._path.indexOf(workingDirectory) == 0) {
+                let relativePackagePath = packagePath.substring(workingDirectory.length);
+                if (relativePackagePath.indexOf(path.sep) == 0) {
+                    relativePackagePath = relativePackagePath.substring(1);
+                }
+                let endIndex = relativePackagePath.indexOf(path.sep);
+                endIndex = endIndex >= 0 ? endIndex : relativePackagePath.length;
+                this._artifactAlias = relativePackagePath.substring(0, endIndex);
```

Comment:
Can the alias contain path-separator as well

Reply:
Alias cannot contain forward or back slash '/' or '\'